easybuild/easyconfigs/g/GSL/GSL-2.6-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'GSL'
+version = '2.6'
+
+homepage = 'https://www.gnu.org/software/gsl/'
+description = """The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers.
+ The library provides a wide range of mathematical routines such as random number generators, special functions
+ and least-squares fitting."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
```

Comment:
@edmondac This easyconfigs should be removed, `GSL` is performance sensitive so we're using `GCC` and `iccifort` as toolchain there (see #9504)

Reply:
Done